README.md

```diff
@@ -2,50 +2,41 @@
-
-Paperless-ngx forked from paperless-ng to continue the great work already done and distribute responsibility among a team of people.
-
-Discussion around that can be found in the issues in the paperless-ng repository:
-[1599](https://github.com/jonaswinkler/paperless-ng/issues/1599)
-[1632](https://github.com/jonaswinkler/paperless-ng/issues/1632)
-## Get in Touch
-
-People interested in continuing the work on paperless-ng(x) and form the organisation connected here on github and created a
-[Matrix Room](https://matrix.to/#/#paperless:adnidor.de) for realtime communication.
+![Paperless logo](https://github.com/paperless-ngx/paperless-ngx/raw/main/resources/logo/web/png/Black%20logo%20-%20no%20background.png#gh-light-mode-only)
+![Paperless logo](https://github.com/paperless-ngx/paperless-ngx/raw/main/resources/logo/web/png/White%20logo%20-%20no%20background.png#gh-dark-mode-only)
 
-# How it Works
-
-Paperless does not control your scanner, it only helps you deal with what your scanner produces.
-
-1. Buy a document scanner that can write to a place on your network.  If you need some inspiration, have a look at the [scanner recommendations](https://paperless-ngx.readthedocs.io/en/latest/scanners.html) page. Set it up to "scan to FTP" or something similar. It should be able to push scanned images to a server without you having to do anything.  Of course if your scanner doesn't know how to automatically upload the file somewhere, you can always do that manually. Paperless doesn't care how the documents get into its local consumption directory.
+<!-- omit in toc -->
+# Paperless-ngx
 
-	- Alternatively, you can use any of the mobile scanning apps out there. We have an app that allows you to share documents with paperless, if you're on Android. See the section on affiliated projects below.
+Paperless-ngx is a document management system that transforms your physical documents into a searchable online archive so you can keep, well, *less paper*.
 
-2. Wait for paperless to process your files. OCR is expensive, and depending on the power of your machine, this might take a bit of time.
-3. Use the web frontend to sift through the database and find what you want.
-4. Download the PDF you need/want via the web interface and do whatever you like with it.  You can even print it and send it as if it's the original. In most cases, no one will care or notice.
+Paperless-ngx forked from [paperless-ng](https://github.com/jonaswinkler/paperless-ng) to continue the great work and distribute responsibility among a team of people. That discussion can be found in issues
+[#1599](https://github.com/jonaswinkler/paperless-ng/issues/1599)
+and [#1632](https://github.com/jonaswinkler/paperless-ng/issues/1632).
 
-Here's what you get:
+#### Demo
+A demo is available at [demo.paperless-ngx.com](http://demo.paperless-ngx.com) using login `demo` / `demo` (content is reset frequently).
```

Comment:
Perhaps update it to be something like this:

A demo is available at [demo.paperless-ngx.com](http://demo.paperless-ngx.com) using login `demo` / `demo` 

> :warning: Warning
> Content is regularly reset and as this is accessible by everyone please do not upload and confidential information.     |

Reply:
Also add demo to the TOC and move it lower down after features.

Reply:
I dont think every heading needs to be in TOC (e.g. this one doesnt), and demo is above TOC on purpose. Its often [one of] the first things you see on an OSS readme. Its like, "enough talking just go play with it".

As for warning about the demo, I dont think we want a warning sign so prominent in the readme, especially if the demo stays here. I think the confidential thing is sorta obvious but cant hurt to mention.

I added a commit for these, lets see how others feel.

Reply:
IMO the demo should stay where it is now, it's a very important feature.